GitExtensionsShellEx/GitExtensionsShellEx.rc

```diff
@@ -26,23 +26,23 @@ LANGUAGE LANG_NEUTRAL, SUBLANG_NEUTRAL
 
 // Icon with lowest ID value placed first to ensure application icon
 // remains consistent on all systems.
-IDI_GITEXTENSIONS       ICON                    "Resources\\git-extensions-logo.ico"
-IDI_ICONADDED           ICON                    "Resources\\IconAdded.ico"
-IDI_ICONBROWSEFILEEXPLORER ICON                    "Resources\\IconBrowseFileExplorer.ico"
-IDI_ICONBRANCHCREATE    ICON                    "Resources\\IconBranchCreate.ico"
-IDI_ICONBRANCHCHECKOUT  ICON                    "Resources\\IconBranchCheckout.ico"
-IDI_ICONREVISIONCHECKOUT ICON                    "Resources\\IconRevisionCheckout.ico"
-IDI_ICONABOUT           ICON                    "Resources\\IconAbout.ico"
-IDI_ICONCLONEREPOGIT    ICON                    "Resources\\IconCloneRepoGit.ico"
-IDI_ICONCOMMIT          ICON                    "Resources\\IconCommit.ico"
-IDI_ICONFILEHISTORY     ICON                    "Resources\\IconFileHistory.ico"
-IDI_ICONPULL            ICON                    "Resources\\IconPull.ico"
-IDI_ICONPUSH            ICON                    "Resources\\IconPush.ico"
-IDI_ICONTRESETFILETO    ICON                    "Resources\\IconResetFileTo.ico"
-IDI_ICONSETTINGS        ICON                    "Resources\\IconSettings.ico"
-IDI_ICONVIEWCHANGES     ICON                    "Resources\\IconDiff.ico"
-IDI_ICONSTASH           ICON                    "Resources\\IconStash.ico"
-IDI_ICONCREATEREPOSITORY ICON                   "Resources\\IconRepoCreate.ico"
+IDI_GITEXTENSIONS           ICON    "..\\Logo\\git-extensions-logo.ico"
```

Comment:
Could you please copy it to Resources folder as well

Reply:
No problem, if you really want to have a copy of it.
I'm afraid, this will make this issue reoccur when the icons will be changed again.